quaint/src/tests/test_api/sqlite.rs

```diff
@@ -83,6 +83,10 @@ impl<'a> TestApi for Sqlite<'a> {
         Quaint::new(CONN_STR).await
     }
 
+    fn create_pool(&self) -> crate::Result<crate::pooled::Quaint> {
+        Ok(crate::pooled::Quaint::builder(CONN_STR)?.build())
```

Comment:
We could probably add a method that returns a `&'static str` connection string and move the implementation to the trait itself to reduce some duplication. No need to change anything, it is totally fine as is and is consistent with `create_additional_connection` and other methods, I'm just thinking out loud and wondering if it might be worth it to de-duplicate a bunch of stuff here generally.

Reply:
I've had the same idea, but I wasn't sure if putting the connection string on the trait leaks too much of an internal detail

Reply:
it would allow us to get rid of a few duplicated implementations though